[2025-04-10T21:15:49.130+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: generate_data.run_generate_script scheduled__2023-01-17T14:00:00+00:00 [queued]>
[2025-04-10T21:15:49.158+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: generate_data.run_generate_script scheduled__2023-01-17T14:00:00+00:00 [queued]>
[2025-04-10T21:15:49.159+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2025-04-10T21:15:49.321+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): run_generate_script> on 2023-01-17 14:00:00+00:00
[2025-04-10T21:15:49.333+0000] {standard_task_runner.py:57} INFO - Started process 5296 to run task
[2025-04-10T21:15:49.353+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'generate_data', 'run_generate_script', 'scheduled__2023-01-17T14:00:00+00:00', '--job-id', '403', '--raw', '--subdir', 'DAGS_FOLDER/dags_1.py', '--cfg-path', '/tmp/tmp98k2430z']
[2025-04-10T21:15:49.377+0000] {standard_task_runner.py:85} INFO - Job 403: Subtask run_generate_script
[2025-04-10T21:15:49.562+0000] {task_command.py:415} INFO - Running <TaskInstance: generate_data.run_generate_script scheduled__2023-01-17T14:00:00+00:00 [running]> on host 7889a671eaba
[2025-04-10T21:15:49.853+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='generate_data' AIRFLOW_CTX_TASK_ID='run_generate_script' AIRFLOW_CTX_EXECUTION_DATE='2023-01-17T14:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-01-17T14:00:00+00:00'
[2025-04-10T21:15:56.594+0000] {logging_mixin.py:151} INFO - Script Output: 
[2025-04-10T21:15:56.596+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-04-10T21:15:56.622+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=generate_data, task_id=run_generate_script, execution_date=20230117T140000, start_date=20250410T211549, end_date=20250410T211556
[2025-04-10T21:15:56.818+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-04-10T21:15:56.854+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
